src/api/erc20/erc20Details.json

```diff
@@ -0,0 +1,4 @@
+{
+  "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359": { "name": "Sai Stablecoin v1.0", "symbol": "SAI", "decimals": 18 },
+  "0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2": { "name": "Maker", "symbol": "MKR", "decimals": 18 }
```

Comment:
So only for mainnet?

Reply:
The likelihood of having the same address for different networks AND that address not pointing to the same token is very low in my opinion.
If it is, I'd say, yes, mainnet has precedence.

Of course, I could just split by network. But is it necessary?

Reply:
nah
:+1: